lua/kubectl/views/jobs/init.lua

```diff
@@ -0,0 +1,80 @@
+local ResourceBuilder = require("kubectl.resourcebuilder")
+local buffers = require("kubectl.actions.buffers")
+local commands = require("kubectl.actions.commands")
+local definition = require("kubectl.views.jobs.definition")
+local informer = require("kubectl.actions.informer")
+local tables = require("kubectl.utils.tables")
+
+local M = {
+  builder = nil,
+}
+
+function M.View(cancellationToken, from_cronjob)
+  if M.builder then
+    M.builder = M.builder
```

Comment:
We can probably rewrite this a bit cleaner using our Resourcebuilder.view, not sure if I'm missing something though?
```
  if M.builder then
    M.builder = M.builder:view(definition, cancellationToken)
  else
    M.builder = ResourceBuilder:new(definition.resource):view(definition, cancellationToken)
  end

```

Reply:
I override the call to self.draw because our draw accepts another parameter (`from_cronjob`).
maybe that's not the smartest way to do it, would love your thoughts